policies/assembler.md

```diff
@@ -0,0 +1,19 @@
+Policy for Accepting Assembler Optimisations
+============================================
+
+New assembler optimisations for any algorithm having a C based implementation in
+any provider are always acceptable (subject to standard review procedures) for
+all platforms in the master branch.
+
+New assembler optimisations are never acceptable for any platform or provider in
+a stable release branch.
```

Comment:
Why does it have the "for any platform or provider" part?

Reply:
To make it clear that it doesn't matter whether we're targeting a new platform or an existing platform, or whether we're targeting default or legacy providers - new assembler optimisations are never allowed. We could simplify the wording to just say "New assembler optimisations are never acceptable in a stable release branch" - but I don't think the current wording is wrong.